### ISSUE TITLE
Arch based dependencies

### ISSUE THREAD

**Issue by Natetronn** [HaikuTeam/animator#1060]
Does anyone know the equivalent Arch based dependencies?

```
sudo pacman -S git curl python2 build-essential libgcrypt20 libcurl4-openssl-dev libssl-dev libgtk-3-0 libgconf-2-4 libnss3
[sudo] password for username: 
warning: git-2.33.0-1 is up to date -- reinstalling
warning: curl-7.79.1-1 is up to date -- reinstalling
warning: python2-2.7.18-4 is up to date -- reinstalling
error: target not found: build-essential
error: target not found: libgcrypt20
error: target not found: libcurl4-openssl-dev
error: target not found: libssl-dev
error: target not found: libgtk-3-0
error: target not found: libgconf-2-4
error: target not found: libnss3
```
_Sorry for the #lazyweb but, figured I'd ask in the hopes of saving some time._

**Comment by FredySandoval** [comment]
```
sudo pacman -S git base-devel libgcrypt openssl gtk3 gconf nss
```
If you had success using it with Figma, let us know how you did. ( I just couldn't make it to work)

**Comment by NikhilSaini38** [comment]
took 458 seconds to perform yarn install around 98% of that time took to build nodegit. after "yarn go" the application started, I don't know if there are any more bugs after that yet.

**Comment by Nitwel** [comment]
> took 458 seconds to perform yarn install around 98% 

Thanks for that info! I thought it was stuck in an infinite loop and quit yarn install after 5 mins.

**Comment by NikhilSaini38** [comment]
> > took 458 seconds to perform yarn install around 98%
> 
> Thanks for that info! I thought it was stuck in an infinite loop and quit yarn install after 5 mins.

Initially, I also thought that and cancelled and retried maybe 3-4 times. Then I just opened htop in a new terminal and watched what node-gyp was doing. It was albeit slow, making progress. So, I just let it do its thing.

**Comment by NikhilSaini38** [comment]
And figma auth probably wont work. I tried.

**Comment by Natetronn** [comment]
> ```
> sudo pacman -S git base-devel libgcrypt openssl gtk3 gconf nss
> ```
> 
> If you had success using it with Figma, let us know how you did. ( I just couldn't make it to work)

Thank you! I'll give a try soon and let you know how I get on.

**Comment by Natetronn** [comment]
Yeah, after messing around with getting the older yarn 1.13.0 working instead of system yarn (there was a newer version installed as a build dependency, so I removed it; now running only nvm versions) I got it to work.

So far, as NikhilSaini38 pointed out, Figma auth via xdg-open isn't working. It opens up the browser window with Figma, after clicking Allow Access and clicking yes to allow xdg-open to do its thing, I just get an error dialog with the following:

`haiku://oauth/figma?code=xxxxxxxxxxxxxxxxxxxx&state=xxxxxxxxxxxxxxxxxxxxxx`

Importing from a URL errored out as well.

ETA

I looked into xdg-open setup and mime types to see if I could get it to connect but, was unsuccessful. I'll have to play with this a bit more later when I have a bit more free time.

**Comment by NikhilSaini38** [comment]
> Yeah, after messing around with getting the older yarn 1.13.0 working instead of system yarn (there was a newer version installed as a build dependency, so I removed it; now running only nvm versions) I got it to work.
> 
> So far, as NikhilSaini38 pointed out, Figma auth via xdg-open isn't working. It opens up the browser window with Figma, after clicking Allow Access and clicking yes to allow xdg-open to do its thing, I just get an error dialog with the following:
> 
> `haiku://oauth/figma?code=xxxxxxxxxxxxxxxxxxxx&state=xxxxxxxxxxxxxxxxxxxxxx`
> 
> Importing from a URL errored out as well.
> 
> ETA
> 
> I looked into xdg-open setup and mime types to see if I could get it to connect but, was unsuccessful. I'll have to play with this a bit more later when I have a bit more free time.

It is probably expects middleware (https://github.com/HaikuTeam/figma-auth) to broker the authentication from figma.
